symfony/client/src/orangehrmPimPlugin/components/FullNameInput.vue

```diff
@@ -76,6 +76,12 @@ export default {
       type: Boolean,
       default: false,
     },
+    middleNameProps: {
+      type: Object,
+      default: () => ({
+        placeholder: 'Middle Name',
+      }),
+    },
```

Comment:
rather than doing like this is it not possible to bind the placeholder attribute to a string prop? Or is the idea that more props can be added if needed?
```suggestion
    middleNamePlaceholder: {
      type: String,
      default: 'Middle Name',
    },
```

Reply:
yes, that idea that more props can be added